intuita-webview/src/codemodList/App.tsx

```diff
@@ -45,12 +44,27 @@ function App() {
 		};
 	}, []);
 
+	const setPublicCodemodsExpanded = useCallback(
```

Comment:
lets move pure functions outside of component scope instead of wrapping in useCallback